biorbd_optim/solver_interface.py

```diff
@@ -5,7 +5,7 @@ def __init__(self):
     def configure(self, **options):
         raise RuntimeError("SolverInterface is an abstract class")
 
-    def solve(self):
+    def solve(self, ocp=None):
```

Comment:
This won't be needed anymore if an ocp reference is conserved by the class. Even though Ipopt doesn't need it specifically, I think it would make sense to do this in the __init__ of the parent class anyway